Getting started for revenue analytics (#12701)
PostHog/posthog.com: contents/docs/revenue-analytics/start-here.mdx

---
title: Getting started with revenue analytics
sidebar: Docs
hideAnchor: true
---

import DataWarehouseSources from './_snippets/data-warehouse-sources'
import { QuestLog, QuestLogItem } from 'components/Docs/QuestLog'

<QuestLog firstSpeechBubble="Let's get started!" lastSpeechBubble="Time to understand your revenue!">

<QuestLogItem 

>     title="Choose your reporting currency"

Reviewer: I feel like connecting a source should go first as it's the most important thing to do and probably has a much higher impact on activation/retention than reporting currency.

    subtitle="Required"

>     icon="IconGear"

Reviewer 2: let's use IconHandMoney!

<img width="118" height="89" alt="image" src="https://github.com/user-attachments/assets/706aba29-a0d4-400c-9d04-335028650e60" />

>
  

>   PostHog will automatically convert revenue from other currencies into your chosen reporting currency. For example, if someone makes a purchase in Canadian Dollars (CAD) and your reporting currency is US Dollars (USD), PostHog converts it into USD to ensure consistency.

Reviewer: ```suggestion
  PostHog automatically converts revenue from other currencies into your chosen reporting currency. For example, if someone makes a purchase in Canadian Dollars (CAD) and your reporting currency is US Dollars (USD), PostHog converts it into USD to ensure consistency.
```

> 

Reviewer: Feels like we need a <br/> or something here, is it always this tight?

<img width="1246" height="628" alt="CleanShot 2025-09-09 at 08 03 53@2x" src="https://github.com/user-attachments/assets/a485b629-0d4b-48d7-9de2-e8cf5a8e0034" />

Reviewer 2: this is due to mdx + gatsby quirks

@gewenyu99 you gotta delete the two spaces that are invisible on line `18` right below `<QuestLogItem>` 😆 to get the right paragraph break

  <ProductScreenshot
      imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/update_currency_light_9345e4640f.png" 
      imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/update_currency_dark_dc5c40bd15.png"
      alt="Revenue analytics currency picker" 
      classes="rounded"
  />

  <CallToAction type="primary" to="https://app.posthog.com/data-management/revenue">
    Configure your reporting currency
  </CallToAction>

</QuestLogItem>

<QuestLogItem
    title="Configure your revenue data source"
    subtitle="Required"
    icon="IconCode2"
>

  You can configure PostHog to integrate with a revenue data source like Stripe, or you can explicitly capture revenue data as PostHog events.

  ### Data warehouse sources

  If your revenue data is already in one of these sources, you can link it to PostHog's data warehouse and start tracking your revenue.

  <DataWarehouseSources />

>   ### Event properties

Reviewer: ```suggestion
  ### Capture revenue events
```

  If you're not using a supported data warehouse source, or you just want to track revenue manually, you can do so by capturing revenue events with revenue properties.

  <CallToAction type="primary" to="/docs/revenue-analytics/capture-revenue-events">
    Capture revenue events
  </CallToAction>

</QuestLogItem>

<QuestLogItem
    title="Visit the revenue analytics dashboard"
    subtitle="Recommended"
    icon="IconGraph"
>

  Once you've configured your revenue data source, you can visit the [revenue analytics dashboard](/docs/revenue-analytics/dashboard) to better understand your revenue.

  <ProductScreenshot
    imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/Screenshot_2025_09_05_at_14_05_35_288e8e0d6c.png"
    imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/Screenshot_2025_09_05_at_14_06_29_e4fe61d678.png"
    alt="Revenue analytics dashboard" 
    classes="rounded"
  />

  <CallToAction type="primary" to="/docs/revenue-analytics/dashboard">
    Visit the revenue analytics dashboard
  </CallToAction>

</QuestLogItem>

<QuestLogItem
  title="Connect to customers"
  subtitle="Recommended"
  icon="IconPeople"
>

>   Linking your revenue data to your customers will allow you to identify your top customers and see how much revenue they're generating.

Reviewer: ```suggestion
  Linking your revenue data to your customers enables you to identify your top customers and see how much revenue they're generating.
```

  When you collect revenue data through events, we'll automatically link the customer to the revenue data. If you're connecting to a payment platform like Stripe, you'll need to manually link the customer to the revenue data.

  <CallToAction type="primary" to="/docs/revenue-analytics/connect-to-customers">
    Connect to customers
  </CallToAction>

</QuestLogItem>

> <QuestLogItem 

Author: @rafaeelaudibert Idk actually know what our pricing plans are please take a look

  title="Use for free"
  subtitle="Try revenue analytics for free while it's in beta"
  icon="IconPiggyBank"
>

  PostHog's revenue analytics is currently in beta, and you can try it for free. 
  
  PostHog is always built to be cost-effective by default, with a generous free tier and transparent usage-based pricing. Our generous free tier means more than 90% of companies use PostHog for free. All of this applies to revenue analytics, too, when it's out of beta.

> 

Reviewer 2: Add a CTA for redundancy since they're at the bottom of the page

```suggestion

    <CallToAction type="primary" to="/docs/revenue-analytics/payment-platforms">
    Connect revenue data
  </CallToAction>

```
